event/README.md

```diff
@@ -3,3 +3,52 @@
 Our `event` module provides common publish/subscribe functionality for events as used at Birdie.
 It specializes pubsub to events, instead of general purposes messages and adds some basic schema
 to the events.
+
+## Metrics
+
+The library also provides metrics both for publishers and subscribers. We use prometheus for metrics.
+The metrics are always sampled but are not registered by default anywhere, so if you want the metrics
+you need to opt-in on them by calling [event.RegisterMetrics](TODO_LINK).
+
+Here we document the metrics generated.
+
+### Publisher
+
+#### event_publish_duration_seconds : histogram
+
+Measure publish duration time for each event.
+Useful when debugging connectivy issues between a service and the message broker.
+
+Labels:
+
+* status : "ack" or "nack".
+* name : name of the event.
+
+#### event_publish_total : counter
+
+Total of published messages.
+
+Labels:
+
+* status : "ack" or "nack".
+* name : name of the event.
+
+### Subscription
+
+#### event_process_duration_seconds : histogram
```

Comment:
Not a huge fan of this metric name x_x.

Reply:
What bothers you? we could use publish/consume WDYT?

Reply:
Not sure what was bothering me..but was feeling off with the name :laughing: . I thought about consume but it also didn't made me happy...maybe nothing will made me happy here :laughing: 

WDYT @lfritz ?

Reply:
This is basically the time it takes until we can ack or nack the message, right? Consume and process both seem fine for that case, in my mind :smile:

Reply:
will leave as process for now